src/test/java/org/opensearch/security/support/Base64HelperTest.java

```diff
@@ -42,7 +44,7 @@ private static final class NotSafeSerializable implements Serializable {
     }
 
     private static Serializable ds(Serializable s) {
-        return deserializeObject(serializeObject(s));
+        return deserializeObject(serializeObject(s, false));
```

Comment:
Nit; Its hard to know what action is going to be taken when passing `true|false` values in as parameters, for this reason I like using enums to clarify the behavior, such as the following example.  What do you think?

```java
serializeObject(s, PackageBehavior.REWRITE_AS_ODFE)
serializeObject(s, PackageBehavior.NONE)
```